packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -613,6 +613,7 @@ export class MessageWidget extends Component {
               {
                 user.get(`currentUser`) &&
                 <ActivityList
+                  newMessagesMessage={formatMessage(messages.newMessagesMessage)}
```

Comment:
I don't understand your message. Maybe you can message me your message and I'll display it in a message window. 😛